### DISCUSSION TITLE
live_endpoint change in boot.cfg buggy

### DISCUSSION THREAD

**Discussion by elsucht** [antonym/netboot.xyz#955] [Q&A]
got netboot.xyz running in a docker container. Webapp is working and I can change the boot.cfg to my needs.
However when I add the line "set live_endpoint http://{mylocalip}:8080" the changes seems to have no effect.
When Im booting my VM with pxe everything seems to work. But it still uses the old live_endpoint, don´t uses local assets, instead downloads all files every time.
What do I miss here?

**Comment by greenaar** [comment]
I'm also seeing this.
